algorithm/sorting/mergesort.cpp

```diff
@@ -6,11 +6,11 @@ void merge(vector<int>&arr, int l, int r){
     // split to smallest piece
     if(l >= r)
         return;
-    int mid = l+(r-l)/2; // avoid a + b > INT_MAX
+    int mid = l+(r-l)/2; //  avoid a + b > INT_MAX
     merge(arr, l, mid);
```

Comment:
Looks like mergesort.cpp already exist.
Your commit message will easily cause to some confusion.